bin.src/genDefectFits.py

```diff
@@ -1,14 +1,12 @@
 #!/usr/bin/env python
 
-import os.path
-import re
-
-import numpy
-import pyfits
 import collections
-
 import lsst.log as log
 from lsst.obs.pfs import PfsMapper
+import numpy
```

Comment:
Elsewhere you import numpy as np.  Please file a ticket to consistently use `import numpy as np` everywhere.